src/main/webapp/WEB-INF/views/cars/drivers/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Car id:<input type="number" name="car_id" required><br>
+    Driver id:<input type="number" name="driver_id" required><br>
```

Comment:
As a say in cars/add.jsp I think it is better to have have a dropdown lists of names not just fields where you put just id.
But in this do as you think better.